tiled/client/composite.py

```diff
@@ -97,6 +98,38 @@ def create_composite(self, key=None, *, metadata=None, specs=None):
         """Composite nodes can not include nested composites by design."""
         raise NotImplementedError("Cannot create a composite within a composite node.")
 
+    def delete_contents(
+        self,
+        keys: Optional[Union[str, Iterable[str]]] = None,
+        external_only: bool = True,
+    ) -> "Composite":
+        """Delete the contents of this Composite node.
+
+        Only constituent arrays or entire tables (with keys listed in `parts`)
+        can be deleted.
+
+        Parameters
+        ----------
+        keys : str or list of str
+            The key(s) to delete. If a list, all keys in the list will be deleted.
+            If None (default), delete all contents.
+        external_only : bool, optional
+            If True, only delete externally-managed data keys. Defaults to True.
```

Comment:
```suggestion
            If True, only delete externally-managed data. Defaults to True.
```

Avoiding "data keys" as an overloaded term (bluesky). And the sentence works fine with just "data".